src/main/java/collections/myHashMap/MyHashMap.java

```diff
@@ -0,0 +1,117 @@
+package collections.myHashMap;
+import collections.MyLinkedList;
+
+import java.util.LinkedList;
+import java.util.Objects;
+
+public class MyHashMap<K, V> {
+    LinkedList<Entry>[] table = new LinkedList[8];
+    public int size = 0;
+
+    public MyHashMap(){}
+
+    public void put(Key key, Value value){
+        if(size >= table.length){
```

Comment:
So, you have an 8 linked lists inside a linked list, if there will be 8, or more elements in all 8 lists, is it right to resize an array?